cli/internal/context/context.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/vercel/turbo/cli/internal/turbopath"
 	"github.com/vercel/turbo/cli/internal/util"
 
-	"github.com/Masterminds/semver"
+	"github.com/Masterminds/semver/v3"
```

Comment:
Newb Go question... why does this import change? We just updated the dependency version right, shouldn't it just take what's in `go.mod`?

Reply:
Go is weird about major versions: https://go.dev/doc/modules/major-version